docker/config/local_dev.env

```diff
@@ -85,22 +85,17 @@ processor.command_pathname=/stackwalk/stackwalker
 
 # Set the telemetry bucket name explicitly
 destination.telemetry.bucket_name=telemetry_bucket
+telemetry.bucket_name=telemetry_bucket
```

Comment:
The `socorro/external/boto/upload_telemetry_schema.py` configman app uses `telemetry.bucket_name` for the name of the bucket to push the schema to. This wasn't set in the local dev environment.

It is set in the server environments, so this change doesn't affect the servers at all.